GoogleAnalyticsTracker.Core/TrackerBase.async.cs

```diff
@@ -63,6 +63,12 @@ private void SetRequired(IGeneralParameters parameters)
         {
             parameters.TrackingId = TrackingAccount;
             parameters.ClientId = AnalyticsSession.GenerateSessionId();
+
+            if (string.IsNullOrWhiteSpace(parameters.UserLanguage))
+            {
+                // Note: another way could be CultureInfo.CurrentCulture
```

Comment:
Typically the server language, but if well developed it would be the client's culture :)